docker/docker-compose.yml

```diff
@@ -50,18 +56,24 @@ services:
       "--no-igd",
       "--hide-my-port",
       "--data-dir=./.localnet/xmr_local/node2",
-      "--p2p-bind-ip=127.0.0.1",
+      "--p2p-bind-ip=0.0.0.0",
+      "--p2p-bind-port=28080",
+      "--confirm-external-bind",
       "--rpc-bind-ip=0.0.0.0",
+      "--rpc-bind-port=28081",
       "--no-zmq",
-      "--confirm-external-bind",
-      "--add-exclusive-node=127.0.0.1:48080",
-      "--rpc-access-control-origins=http://localhost:8080",
+      "--add-exclusive-node=node1:48080",
+      "--rpc-access-control-origins='*'",
```

Comment:
Any reason to open up the access control origins from http://localhost:8080? Thinking to keep it as is to be more representative of production.